client/core/simnet_trade.go

```diff
@@ -90,7 +91,7 @@ type assetConfig struct {
 	isToken          bool
 }
 
-var testLookup = map[string]func(s *simulationTest) error{
+var TestLookup = map[string]func(s *simulationTest) error{
```

Comment:
This is a little awkward since it's exporting a func that takes an unexported type (`simulationTest`), but this is all simnet bidness, so I don't much care.  It's just unfortunate to have this kind of stuff end up in `core`'s public API.

Reply:
Exporting a getter now if that's better?